servicecontrol/servicecontrol-in-practice.md

```diff
@@ -1,35 +1,70 @@
 
-##### Less can be more
+Here are some important considerations when installing and deploying ServiceControl for better performance.
 
-Moving forward, take a thoughtful approach in the adoption of plugins and features:
 
-- Read the [Capacity Planning Guide](/servicecontrol/capacity-and-planning.md) for ServiceControl.
-- Read the [Troubleshooting Guide](/servicecontrol/troubleshooting.md) before deployment in production.
-- [Turn off auditing](/nservicebus/operations/auditing.md) on all endpoints as well as [heartbeats and custom checks](/servicepulse/how-to-configure-endpoints-for-monitoring.md).
-- Perform load tests to baseline the solution.
-- When comfortable with the performance of the system try adding [Heartbeats](/servicepulse/intro-endpoints-heartbeats.md) to monitor the system again.
-- Try increasing the [heartbeat interval](/servicecontrol/plugins/heartbeat.md). Ideally heartbeat updates should not occur more frequently than ServiceControl can process them or more than Operations staff are prepared to monitor them with ServicePulse.
-- With each additional change perform a load test again adjusting the heartbeat interval satisfied with the result.
-- Repeat the process of considering the business relevance, system impact and load testing with each [Custom Check](/servicecontrol/plugins/custom-checks.md).
-- Repeat the process of considering the business relevance, system impact and load testing with each Endpoint [Audit](/nservicebus/operations/auditing.md).
+## Hardware & Installation Considerations:
 
-##### Tips and Tricks
+- Read the [Capacity Planning](/servicecontrol/capacity-and-planning.md) and [Troubleshooting](/servicecontrol/troubleshooting.md) guides for ServiceControl.
+- Each environment should have a dedicated ServiceControl instance.
+- Other applications or NServiceBus endpoints running on the same machine as ServiceControl can compete for hardware resources and therefore negatively impact ServiceControl's performance. For optimal performance run SC on a dedicated machine.
+- If ServiceControl is installed on a Virtual Machine, ensure the machine is capable of high levels of I/O traffic. The amount of I/O required depends on the system being monitored. Run performance 
+tests with realistic message loads to baseline hardware requirements.
 
-- If running anti-virus software exclude the ServiceControl [database directory](/servicecontrol/configure-ravendb-location.md) from virus checks.
-- The recommended approach is that customers don't downgrade major and minor releases of ServiceControl. ServiceControl uses an embedded database and changes to the internal data structures can occur between releases. Rolling back may cause index corruption or data loss. The recommended approach is that customers perform testing in a lower environment before upgrading in production environments.
\ No newline at end of file
+
+## Message Throughput Considerations
```

Comment:
Message throughput also included how many messages you want to be able to retry. We should mention that we need to reserve capacity for the IO involved when doing large groups of retries.